packages/debug/src/browser/editor/debug-editor-model.ts

```diff
@@ -216,13 +216,15 @@ export class DebugEditorModel implements Disposable {
         const column = breakpoint.column;
         const range = typeof column === 'number' ? new monaco.Range(lineNumber, column, lineNumber, column + 1) : new monaco.Range(lineNumber, 1, lineNumber, 1);
         const { className, message } = breakpoint.getDecoration();
+        const firstNonWhitespaceColumn = 1;
```

Comment:
but it should no be no 1, but computed as in link which I sent yesterday, i.e. first non whitespace character